linkerd/app/outbound/src/http/logical/policy/route/backend.rs

```diff
@@ -0,0 +1,102 @@
+use super::{super::Concrete, filters};
+use linkerd_app_core::{proxy::http, svc, Error, Result};
+use linkerd_http_route as http_route;
+use linkerd_proxy_client_policy as policy;
+use std::{fmt::Debug, hash::Hash, sync::Arc};
+
+#[derive(Debug, PartialEq, Eq, Hash)]
+pub(crate) struct Backend<T, F> {
+    pub(crate) concrete: Concrete<T>,
+    pub(crate) filters: Arc<[F]>,
+}
+
+pub(crate) type Matched<T, M, F> = super::Matched<M, Backend<T, F>>;
```

Comment:
TIOLI: maybe this ought to be `MatchedBackend`, for consistency with `MatchedRoute`?